examples/convert_eeg_to_bids.py

```diff
@@ -175,8 +175,8 @@
 event_id = {'rest': 0}
 
 # Now convert our data to be in a new BIDS dataset.
-bids_basename = BIDSPath(subject=subject_id, task=task)
-write_raw_bids(raw, bids_basename, bids_root, event_id=event_id,
+bids_path = BIDSPath(subject=subject_id, task=task)
+write_raw_bids(raw, bids_path, bids_root, event_id=event_id,
```

Comment:
we still need to pass `bids_root`? maybe time to deprecate that parameter?

Reply:
I agree. Shall I do that in a next round? I think this one being purely cosmetic has a large diff, so would prefer just getting it out of the way, so the refactoring of kwarg arguments is small diff and easy to review.